docs/en/users/02_First_steps.md

```diff
@@ -2,7 +2,7 @@ Learning how to handle a new application is not always easy. We build FreshRSS t
 
 This section guides you to the pages you need as a new comer.
 
-[After you installed the application](01_Installation.md), the first step is to add one or more feeds. You have few options:
+[After you installed the application](../admins/02_Installation.md), the first step is to add one or more feeds. You have few options:
```

Comment:
After installing the application

How about "the first step is to add some feeds"?